### PR TITLE
Restyled HUD to match redesigns

### PR DESCRIPTION
**Pull request by davidkpiano**
For #17: https://github.com/Glimpse/Glimpse.Design/blob/master/%5BLatest%5D%20HUD.md

- Added styles to match existing structure with designs
- Commented out some irrelevant parts that are not visible in the updated designs
- Added a link to Glimpse (per the design)
- Disabled popup views
- (WIP) Blurred background
screen shot 2017-02-16 at 3 03 10 pm
screen shot 2017-02-16 at 3 03 20 pm

### CODE REVIEW

**Comment by mike-kaufman** [comment]
can you include a screenshot?

File: src/index.scss [hunk 1]
```diff
@@ -41,48 +73,45 @@
...
-    position: absolute;
-    color: white;
-    text-transform: uppercase; 
-    cursor: pointer;
-    font-weight: bold;
-    height: 100%;
-    width: 12px;
-    left: -12px;
+    display: none;
+    // position: absolute;
```
Comment (mike-kaufman) [inline src/index.scss]: is there a reason this commented-out stuff is still here?  CAn you delete it?

File: src/sections/section-http.js [hunk 2]
```diff
@@ -34,9 +34,9 @@ var structure = {
     layout: {
         mini: {
             request: {},
-            wire: {},
-            server: {},
-            client: {}
+            // wire: {},
```
Comment (mike-kaufman) [inline src/sections/section-http.js]: can you add a comment noting these are temporarily disabled?

File: src/sections/section.js [hunk 3]
```diff
@@ -28,5 +28,5 @@ module.exports = {
 
 // TODO: Need to come up with a better self registration process
 require('./section-http');
-require('./section-host');
-require('./section-ajax');
+// require('./section-host');
```
Comment (mike-kaufman) [inline src/sections/section.js]: can you add comment noting these are temporarily disabled.

File: src/sections/section-http.js [hunk 4]
```diff
@@ -11,7 +11,7 @@ var structure = {
     title: 'HTTP',
     id: 'http',
     color: '#e2875e',
-    popup: {
+    xpopup: {
```
Comment (mike-kaufman) [inline src/sections/section-http.js]: why did this change from `popup` to `xpopup`?

File: src/sections/util/rendering.js [hunk 5]
```diff
@@ -26,7 +26,7 @@ var section = function(structure, details, opened) {
     }
     html += '</div>';
 
-    if (!structure.popup.suppress) { html += popup(structure, details); }
+    // if (!structure.popup.suppress) { html += popup(structure, details); }
```
Comment (mike-kaufman) [inline src/sections/util/rendering.js]: can you add a comment saying this is temporarily disabled?

**Review by mike-kaufman — CHANGES_REQUESTED** [review]
some questions/comments.

File: src/sections/section-http.js (hunk 4 above)
Comment (davidkpiano) [reply src/sections/section-http.js]: Ah, meant to comment that out - can revert.

**Comment by davidkpiano** [comment]
@mike-kaufman Comments addressed; LGTY?

**Review by mike-kaufman — APPROVED** [review]

**Comment by mike-kaufman** [comment]
lgtm

**Pull request merged**